### PR TITLE
Install the bzImage for kernel-uvm

### PR DESCRIPTION
**Pull request by dallasd1**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Kernel-uvm builds the bzImage but it is not installed as part of its RPM. This PR installs the already built bzImage. This binary is used as input to generate IGVM files for confidential containers.

###### Change Log  

- Install bzImage

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Pipeline build id: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=355246&view=results
- Pipeline build id (arm): https://dev.azure.com/mariner-org/mariner/_build/results?buildId=355225&view=results

### CODE REVIEW

File: SPECS/kernel-uvm/kernel-uvm.spec [hunk 1]
```diff
@@ -137,6 +137,9 @@ cp .config %{buildroot}%{_prefix}/src/linux-headers-%{uname_r} # copy .config ma
 ln -sf "%{_prefix}/src/linux-headers-%{uname_r}" "%{buildroot}/lib/modules/%{uname_r}/build"
 find %{buildroot}/lib/modules -name '*.ko' -exec chmod u+x {} +
 
+mkdir -p %{buildroot}/boot/kernel-uvm/
```
Comment (ms-mahuber) [inline SPECS/kernel-uvm/kernel-uvm.spec:140]: variable for kernel-uvm? ${name} or $uname_r} folder?

File: SPECS/kernel-uvm/kernel-uvm.spec [hunk 2]
```diff
@@ -137,6 +137,9 @@ cp .config %{buildroot}%{_prefix}/src/linux-headers-%{uname_r} # copy .config ma
...
+install -vm 600 arch/%{arch}/boot/bzImage %{buildroot}/boot/kernel-uvm/bzImage
```
Comment (ms-mahuber) [inline SPECS/kernel-uvm/kernel-uvm.spec:141]: variable for bzImage? something similar to %{image_fname}?

File: SPECS/kernel-uvm/kernel-uvm.spec [hunk 3]
```diff
@@ -145,13 +148,17 @@ find %{buildroot}/lib/modules -name '*.ko' -exec chmod u+x {} +
 %ifarch x86_64
 /lib/modules/%{name}/vmlinux
 %endif
+/boot/kernel-uvm/bzImage
```
Comment (ms-mahuber) [inline SPECS/kernel-uvm/kernel-uvm.spec:151]: Thought: to potentially reconsider about location for this file to drop. the vmlinux file is being dropped under the CLH folder. potentially we want to align here?

Comment (Camelron) [reply SPECS/kernel-uvm/kernel-uvm.spec:151]: If the UVM kernel doesn't need to be in the boot partition I think it makes sense to move it. The host boots from /boot, UVM is more like a component of kata-shim.

File: SPECS/kernel-uvm/kernel-uvm.spec (hunk 2 above)
Comment (dallasd1) [reply SPECS/kernel-uvm/kernel-uvm.spec]: Binary moved out of /boot and into same directory as vmlinux.bin

File: SPECS/kernel-uvm/kernel-uvm.spec (hunk 3 above)
Comment (dallasd1) [reply SPECS/kernel-uvm/kernel-uvm.spec]: Binary moved out of /boot and into same directory as vmlinux.bin

File: SPECS/kernel-uvm/kernel-uvm.spec (hunk 1 above)
Comment (dallasd1) [reply SPECS/kernel-uvm/kernel-uvm.spec]: New directory removed and binary placed with vmlinux.bin

File: SPECS/kernel-uvm/kernel-uvm.spec (hunk 2 above)
Comment (dallasd1) [reply SPECS/kernel-uvm/kernel-uvm.spec]: I don't think this filename changes at all, so not sure if a variable is necessary for this one.

**Review by ms-mahuber — APPROVED** [review]

**Pull request merged**